doc/source/whatsnew/v0.18.2.txt

```diff
@@ -24,10 +24,13 @@ New features
 
 
 
+.. _whatsnew_0182.enhancements.other:
 
 
+Other enhancements
+^^^^^^^^^^^^^^^^^^
 
-
+- Add ``errors`` flag to tz_localize, so you can silently ignore nonexistent timestamps and replace them with ``NaT`` if ``errors`` is set to ``'coerce'``. The default behaviour is still raising a NonExistentTimeError (``errors='raise'``)
```

Comment:
double backticks on ``tz_localize`` and ``NonExistenTimeError``. add the issue number as well.